homeassistant/components/sensor/fail2ban.py

```diff
@@ -64,7 +64,7 @@ def __init__(self, name, jail, log_parser):
         self._name = '{} {}'.format(name, jail)
         self.jail = jail
         self.ban_dict = {STATE_CURRENT_BANS: [], STATE_ALL_BANS: []}
-        self.last_ban = None
+        self.last_ban = ''
```

Comment:
I don't see how this would solve the bug. The `update` method is run before the first state update of the entity, cause `add_devices` is called with second argument `True`. There will always be at least one state update of a new entity.

Reply:
add to clear out....restarted fail2ban and log is 
```2017-11-19 11:59:05,230 fail2ban.actions        [13398]: NOTICE  [hass-iptables] Ban 107.77.201.xxx``` and shows up in sensor ... after restart ```Ha ~ # grep -E 'Ban|Unban' /var/log/fail2ban.log
2017-11-19 11:59:05,230 fail2ban.actions        [13398]: NOTICE  [hass-iptables] Ban 107.77.201.xxx``` and sensor is empty

Reply:
There is no initial state for the very first state update.